utilities/tools/Clear-ManagementGroupDeployment.ps1

```diff
@@ -0,0 +1,96 @@
+﻿
```

Comment:
We should probably add a wiki page for this as done for other utilities.

Reply:
Was thinking about this - but I couldn't find a place where to put it. It was easy with the original Wiki structure as we had the 'utilities' parent folder, but now?
I eventually ditched the idea as I figured that we'll eventually invoke it via a scheduled pipeline in the other issue and it's not like we documented the other pipeline scripts ^^. Thoughts?

Reply:
I think it naturally belongs to the CI environment, deployment validation in particular. It makes sense to be used only after deploying modules at mg scope.

Reply:
Alternatively - Known issues...

Reply:
Added